docs/Database-Structure.md

```diff
@@ -329,6 +332,9 @@ CREATE TABLE pa_application_callback
 | type           | VARCHAR(64) | - | Callback type: `ACTIVATION_STATUS_CHANGE` or `OPERATION_STATUS_CHANGE`.   |
 | attributes     | TEXT | - | Callback attributes as a key-value map, serialized into JSON.             |
 | authentication | TEXT | - | Callback HTTP request authentication configuration, serialized into JSON. |
+| max_attempts | INTEGER | - | Maximum number of attempts to dispatch a callback. |
```

Comment:
is there any expected/recommended value for infinity repetion?

Reply:
We could recommend to use a very large number. An alternative is to implement the logic based on some negative constant, such as `-1`.

Reply:
Why would we allow infinity repetition? It does not seem to be a good idea.